dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -779,6 +779,10 @@ ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to create a JSON file at
 ERROR_SINGLE_REPO_CONNECTION_FAILED.explanation=The feature in the ESA file will not be installed.
 ERROR_SINGLE_REPO_CONNECTION_FAILED.useraction=Ensure that write access is available for the directory where the JSON file will be created.
 
+MSG_ACCEPT_LICENSE_REQUIRED=CWWKF1390E:The following feature(s) require the above license(s) to be accepted: {0}.
+MSG_ACCEPT_LICENSE_REQUIRED.explanation=The feature(s) will not be installed until the license(s) are accepted.
+MSG_ACCEPT_LICENSE_REQUIRED.useraction=Please accept or reject the corresponding license(s).
```

Comment:
I would recommend removing the word "please" so that the useraction reads "Accept or reject the corresponding license(s)."